bindings/python/examples/how_tos/nft_collection/01_mint_collection_nft.py

```diff
@@ -65,10 +65,11 @@ def get_immutable_metadata(index: int) -> str:
     transaction = wallet.mint_nfts(chunk)
 
     # Wait for transaction to get accepted
```

Comment:
Any point in keeping all these comments?

Reply:
Probably not